tests/helper/helper_dev.go

```diff
@@ -121,25 +123,20 @@ func (o DevSession) Stop() {
 	o.session.Interrupt()
 }
 
-// RunDevMode takes a regExp that it passes to StartDevMode to find matches in the output of its session structure
-// It runs a dev session and executes the `inside` code when the dev mode is completely started.
-// The inside handler is passed the internal session pointer
-// and the contents of the standard and error outputs at the time the handler is called
-func RunDevMode(regExp string, inside func(session *gexec.Session, outContents []byte, errContents []byte, matches []string)) error {
-	session, outContents, errContents, matches, err := StartDevMode(regExp)
+// RunDevMode runs a dev session and executes the `inside` code when the dev mode is completely started
+// The inside handler is passed the internal session pointer and the contents of the standard and error outputs at the time the handler is called
+func RunDevMode(inside func(session *gexec.Session, outContents []byte, errContents []byte, urls []string)) error {
+	session, outContents, errContents, urls, err := StartDevMode()
 	if err != nil {
 		return err
 	}
 	defer session.Stop()
-	inside(session.session, outContents, errContents, matches)
+	inside(session.session, outContents, errContents, urls)
 	return nil
 }
 
 // FindAllMatchingStrings returns all matches for the provided regExp as a slice of strings
 func FindAllMatchingStrings(s, regExp string) []string {
-	if regExp == "" {
-		return nil
-	}
 	regExp = fmt.Sprintf("%s.*", regExp)
```

Comment:
You don't need to add `.*` at the end of the regexp

Reply:
I am clearly in a rush to get this PR in. Or may I say, in a rush to get rid of this PR. 😄

Reply:
That entire line could be gotten rid of.

Reply:
Done d4ee0f5.